addons/point_of_sale/models/pos_order.py

```diff
@@ -253,6 +255,8 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
     session_move_id = fields.Many2one('account.move', string='Session Journal Entry', related='session_id.move_id', readonly=True, copy=False)
     to_invoice = fields.Boolean('To invoice')
     is_invoiced = fields.Boolean('Is Invoiced', compute='_compute_is_invoiced')
+    is_tipped = fields.Boolean('Is this already tipped?')
```

Comment:
shouldn't we put the field "readonly" like for tip_amount

Reply:
I agree.